frontend/src/types/search/searchRequestTypes.ts

```diff
@@ -12,7 +14,7 @@ export type PaginationOrderBy =
   | "opportunity_number"
   | "opportunity_title"
   | "agency_name"
-  | "top_level_agency_code"
+  | "top_level_agency_name"
```

Comment:
seems to be a change made in the data that was not updated in the frontend typing